File: assets/rest/samples/python/AccountsAPI.getDelegates.get.py
from risesdk import APIError, Client

api = Client('https://wallet.rise.vision/api/')
try:
    delegates = api.accounts.get_account_delegates('8093718274007724701R')
    print(delegates[0].public_key.hex())
except APIError as err:
    print('API error:', err)

File: assets/rest/samples/elixir/AccountsAPI.getDelegates.get.exs
import RiseElixir

try do
  result = RiseElixir.Accounts.get_delegates(%{address: "8093718274007724701R"})
  IO.puts("Delegate publicKey: #{Enum.at(result["delegates"], 0)["publicKey"]}")
  # Delegate publicKey: aaabdbb92c4bd64b175c6228f3a868ca2bbe8ff034d160364bf3e2f448148aa3
catch
  err -> "API error #{err}"
end
